src/main/java/weizberg/citibike/StationMethods.java

```diff
@@ -0,0 +1,99 @@
+package weizberg.citibike;
+
+import io.reactivex.rxjava3.core.Single;
+import io.reactivex.rxjava3.schedulers.Schedulers;
+import weizberg.citibike.json.DataCollection;
+import weizberg.citibike.json.Station;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
```

Comment:
Instead of just making this a class with a bunch of methods, why don't you make this into some sort of functional class with member variables